### ISSUE TITLE
Builds on OTP 17 are broken

### ISSUE THREAD

**Issue by g-andrade** [g-andrade/locus#5]
```
==> Verifying dependencies...
===> The registry repository hexpm uses a record format that has been deprecated for security reasons. The repository should be updated in order to be safer. You can disable this check by setting REBAR_NO_VERIFY_REPO_ORIGIN=1
===> Package not found in any repo: getopt 1.0.1
make: *** [cli] Error 1
The command "RUNNING_ON_TRAVIS=yes make travis_test" exited with 2.
```

**Comment by g-andrade** [comment]
rebar3 3.11.0 has [removed support](https://github.com/erlang/rebar3/releases/tag/3.11.0) for OTP 17 - this is the reason why Travis CI execution of the test cases is failing on it, as the latest rebar3 official build is fetched every time.

**Comment by g-andrade** [comment]
Won't fix; support for OTP 17 removed on `1.7.0`.
